Idno/Common/Page.php

```diff
@@ -864,7 +864,7 @@ function setEntity($entity) {
          * @param $entity
          * @return object|null
          */
-        function getEntity($entity) {
+        function getEntity() {
```

Comment:
If this is an object|null, consider return type hinting - e.g.  :  ? BaseObject

Reply:
Docs are good, but also consider using the language construct, e.g.

```
use Idno\Common\Entity;

...

function getEntity($entity) : ? Entity {

...

}

```

Reply:
Done, although now I've done it, I don't really like how the file is now inconsistent. I'll make a note to go back and bring it all into line, but we should probably make a rule to keep files internally consistent. It would be inappropriate to do that in this PR, but that's something for another time.

Reply:
(Docs weren't good - phpdoc doesn't support this syntax. I've changed it back but added return type hinting.)